docs/releases/6.0.txt

```diff
@@ -162,6 +162,13 @@ Minor features
 
 * The new :attr:`.AdminSite.password_change_form` attribute allows customizing
   the form used in the admin site password change view.
+* When :attr:`.ModelAdmin.list_select_related` is ``False`` (the default),
```

Comment:
reminder: 6.1

Also, I think we will need to update this slightly:
https://github.com/django/django/blob/8b84364d469e394d9f04b4f96a7da1fc16d93fce/docs/misc/design-philosophies.txt#L129-L131